Assets/Scripts/ScriptingSystem/FunctionBlocks/DebugN.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DebugN : FunctionBlock
+{
+    [SerializeField] private int number = 0;
```

Comment:
No need to manually set 0, the compiler will do this automatically since 0 is default